test/rcd_test/Rakefile

```diff
@@ -41,11 +41,16 @@ namespace "gem" do
 
       desc "Build the native gem for #{plat}"
       task plat => 'prepare' do
+        config = if ENV['RCD_TEST_CONFIG']
```

Comment:
Is this condition necessary? An ending "--" to the rake task shouldn't disturb.

Reply:
Yup, you're right, I'll simplify.